### PR TITLE
Fix formatting of markdown.

### PR DESCRIPTION
**Pull request by nabil-boag**
Sorry about the big pull request. The markdown numbering is all screwed up because of some white space issues.

I also removed a random div element from the code and added some asterisks to a couple of items.

### CODE REVIEW

File: src/main/markdown/doc/latest/tutorial/codeclient.md [hunk 1]
```diff
@@ -26,26 +26,26 @@ You need a data structure to hold the list of stock symbols the user has entered
...
-      private VerticalPanel mainPanel = new VerticalPanel();
-  private FlexTable stocksFlexTable = new FlexTable();
-  private HorizontalPanel addPanel = new HorizontalPanel();
-  private TextBox newSymbolTextBox = new TextBox();
-  private Button addStockButton = new Button("Add");
-  private Label lastUpdatedLabel = new Label();
-  private ArrayList<String> stocks = new ArrayList<String>();
     
-```
+    ```
```
Comment (jDramaix) [inline src/main/markdown/doc/latest/tutorial/codeclient.md]: This doesn't fix the problem. The markdown engine has some difficulties to handle triple backticks inside lists. The best is to remove the triple backticks and just indent the block by 4*(n +1) (n being the depth of the list)

File: src/main/markdown/doc/latest/tutorial/codeclient.md [hunk 2]
```diff
@@ -26,26 +26,26 @@ You need a data structure to hold the list of stock symbols the user has entered
...
+        
+    ```
 
 2.  Eclipse flags ArrayList and suggests you include the import declaration.
 3.  Include the import declaration.
-
-```
-import java.util.ArrayList;
-```
+
```
Comment (jDramaix) [inline src/main/markdown/doc/latest/tutorial/codeclient.md]: idem indent the code block by 8 spaces

File: src/main/markdown/doc/latest/tutorial/codeclient.md (hunk 1 above)
Comment (jDramaix) [inline src/main/markdown/doc/latest/tutorial/codeclient.md]: Follow this rule for each block of code

**Comment by jDramaix** [comment]
Thanks for fixing that. It's one of the big issue I've introduced when I've converted everything to markdown

File: src/main/markdown/doc/latest/tutorial/codeclient.md (hunk 2 above)
Comment (nabil-boag) [inline src/main/markdown/doc/latest/tutorial/codeclient.md]: Thanks @jDramaix fixed with 3e3e5e08a13d9e97defcc721bd2406a407957d62

**Comment by nabil-boag** [comment]
Thanks for the feedback @jDramaix please check out 3e3e5e08a13d9e97defcc721bd2406a407957d62

File: src/main/markdown/doc/latest/tutorial/codeclient.md [hunk 3]
```diff
@@ -141,89 +130,80 @@ Timer is a single-threaded, browser-safe timer class. It enables you to schedule
...
 
-```
+            import com.google.gwt.user.client.Timer;
 
-import com.google.gwt.user.client.Timer;
-```
 3.  Specify the refresh rate.
     *  If you are using Eclipse shortcuts, select `Create constant 'REFRESH_INTERVAL'` then specify the refresh interval in milliseconds, `5000`.
     *  Otherwise, just cut and paste from the highlighted code below.
     *
```
Comment (jDramaix) [inline src/main/markdown/doc/latest/tutorial/codeclient.md]: Could you remove this star ?

**Comment by jDramaix** [comment]
That's better ! Thank you very much !! 

Could you fix my last comment ? It's not related to your change but as you are editing this page, you could easily do it.

**Comment by nabil-boag** [comment]
@jDramaix Updated code with your latest comment here 1e43f9f9bef0163505eae0aa0cfe0c4f1a3956c6

**Comment by jDramaix** [comment]
Thanks

**Pull request merged**